sklearn/feature_selection/_univariate_selection.py

```diff
@@ -230,19 +230,16 @@ def chi2(X, y):
 
 
 @_deprecate_positional_args
-def f_regression(X, y, *, center=True):
-    """Univariate linear regression tests.
+def r_regression(X, y, *, center=True):
+    """Univariate linear regression tests returning Pearson R.
 
     Linear model for testing the individual effect of each of many regressors.
     This is a scoring function to be used in a feature selection procedure, not
     a free standing feature selection procedure.
 
-    This is done in 2 steps:
-
-    1. The correlation between each regressor and the target is computed,
-       that is, ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) *
-       std(y)).
-    2. It is converted to an F score then to a p-value.
+    The cross correlation between each regressor and the target is computed,
+    that is, ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) *
+    std(y)).
 
```

Comment:
Maybe add a comments such as:

`r_regression` can be used to rank features as a slightly cheaper alternative to `f_regression` if p-values are not needed. Note however that `r_regression` values can be positive or negative.

`abs_r_regression` can be used to rank features by correlation magnitude instead.

Reply:
I've added a such comments in the docstring of `f_regression` accordingly.
I guess `abs_r_regression` is the main alternative to it for variable selection.

I also have slightly rectified the docstring of `r_regression`